packages/babel-traverse/src/path/introspection.js

```diff
@@ -398,3 +398,44 @@ export function _resolve(dangerous?, resolved?): ?NodePath {
     }
   }
 }
+
+export function isConstantExpression() {
```

Comment:
I know this is different than evaluate but do we have to do this for other kinds of nodes? https://github.com/babel/babel/blob/master/packages/babel-traverse/src/path/evaluation.js#L78

Reply:
@hzoo i probably have skipped some kinds of nodes, just included what i could think at the moment - thanks for the pointer to the `_evaluate`, I might expand this `isConstantExpression` (btw is it good name?) later with info found in the `_evaluate`